docs/API.md

```diff
@@ -108,7 +108,7 @@ Default: `0777`
 
 Octal permission string specifying mode for any folders that need to be created for output folder.
 
-### gulp.task(name[, deps], fn)
+### gulp.task(name [, deps] | [, deps[, fn]] | [, fn])
```

Comment:
I'm confused by the function signature you changed this to I don't think we support an array of deps as the third argument